assets/base.css

```diff
@@ -249,6 +249,10 @@ html.no-js .no-js-hidden {
   padding: 0 1.5rem;
 }
 
+body:has( .section-header .drawer-menu) .announcement-bar-section slideshow-component {
```

Comment:
To summarize some thoughts from our pairing on this topic.. I'm actually curious if @YoannJailin thinks it's reasonable to remove the page-width limitation on the slider for _all_ menu types, not just drawer. Particularly when you consider the possibility of adding more content to the utility bar in the future. This would also come with the minor benefit of avoiding the use of :has and the possibility of different results in different browsers.

**Current** (with dropdown or mega menu):
<img src="https://screenshot.click/23-53-z00ew-xyfia.png" width="600" />

**Proposed** (all):
<img src="https://screenshot.click/23-55-z3y29-jgc9j.png" width="600" />

Reply:
Hey @kmeleta, sorry i miss your comment. Usually i am always up for solutions that ease everyone's work! But I think we should stick on aligning the Bar content on the grid when Menu is set to dropdown or mega menu vs full width when it's a drawer on desktop. I am okay if a few users don't see the bar full width when menu is set to drawer. From the numbers i see on browsers that won't show the best behavior, the impact is minimal. I believe the result of having a full width top bar vs an on grid header would look misaligned.

Reply:
I had a meeting with Yoann. From a UX perspective we still want to keep the current logic for the utility-bar and follow the header width. The reason is that for the wide screens it may look bad. Also, for the future when we consider adding more content there will be a bit different layout inside the utility-bar (it's presented on Figma).

We are agreed to keep` :has` for this case since the most popular browsers supports it and even those that don't won't have issues with functionality.